file/lib/file_source_endpoint.ex

```diff
@@ -1,15 +1,18 @@
-defmodule Membrane.RTC.Engine.Support.FileSourceEndpoint do
-  @moduledoc false
-
-  # Endpoint that publishes data from a file.
-  # Starts publishing data on receiving `:start` message.
+defmodule Membrane.RTC.Engine.Endpoint.File do
+  @moduledoc """
+  An Endpoint responsible for publishing data from a file (Currently supports only OPUS and H264).
```

Comment:
I think we should mention about `after_source_transformation` option in case someone wants to read something other than OPUS or H264